pkgs/top-level/python-packages.nix

```diff
@@ -26937,4 +26937,113 @@ in modules // {
       license = licenses.wtfpl;
     };
   };
+
+  uptime = buildPythonPackage rec {
+    name = "uptime-${version}";
+    version = "3.0.1";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "Cairnarvon";
+      repo = "uptime";
+      rev = "1ddfd06bb300c00e6dc4bd2a9ddf9bf1aa27b1bb";
+      sha256 = "148rqj0asw65c3z4whnqphfmkhlm181p296b7pxc1x79n97gn1pf";
+    };
+
+    meta = {
+      description = "Cross-platform uptime library for the Python";
+      homepage = "https://github.com/Cairnarvon/uptime";
+    };
+  };
+
+  pytestHttpbin = buildPythonPackage rec {
+    name = "pytest-httpbin-${version}";
+    version = "0.2.0";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "kevin1024";
+      repo = "pytest-httpbin";
+      rev = "ae314b2a18eecac2a9d3b2034bfede9805ce7907";
+      sha256 = "0j5rcqlh59sqcv6csnk0i73xi5p43h32ynnz550q8pv9q68lzqa2";
+    };
```

Comment:
same here https://pypi.python.org/pypi/pytest-httpbin/0.2.3
Or is the specific version required?
In that case https://pypi.python.org/pypi/pytest-httpbin/0.2.0